assets/sendmail_template.txt

```diff
@@ -59,4 +59,29 @@ Content-Disposition: inline; filename="NGI_logo.png"
   flatten().
   join( '\n' ) %>
 
---ngimimeboundary--
+<%
+if (mqcFile){
+def mqcFileObj = new File("$mqcFile")
+if (mqcFileObj.length() < 25000000){
```

Comment:
Damn, you're good! When I saw the email I was building up to ask you to add a filesize check for the report attachment 😆 You beat me to it!

![lt3n5n](https://user-images.githubusercontent.com/465550/40491117-84141bcc-5f6d-11e8-8d8e-0e6f91945c50.jpg)

Reply:
ok, what would be even cooler is if this is a `params.maxMultiqcEmailFileSize` parameter so that it can be overwritten easily...

Reply:
I like the idea 👍 I am confused in where to have the default `params.maxMultiqcEmailFileSize = 25000000`, in `main.nf` or `base.conf` ?

Reply:
Technically it could be either, but I'm trying to put these in the config going forward, so `base.conf` please 👍